post/src/main/resources/application.yml

```
@@ -0,0 +1,45 @@
+    password: ${DB_PW}
+
+  jpa:
+    database: mysql
+    database-platform: org.hibernate.dialect.MySQLDialect
+    show-sql: true
+    hibernate:
+      ddl-auto: update
+    properties:
+      hibernate:
+        format_sql: true
+        #default_batch_fetch_size: 100
```

YAMLlint (1.38.0)

[warning] missing starting space in comment

(comments)